hands-on-exercise/2-ignite-cli-adv/3-game-fifo.md

```diff
@@ -188,6 +188,23 @@ How do you implement a FIFO from which you extract elements at random positions?
         }
     ```
 
+6. Adjust the types genesis test, so that it incorporates the newly added fields FifoHeadIndex and FifoTailIndex with the correct initialization values.
```

Comment:
I would put it lower within the unit tests section.

Reply:
moved!